### ISSUE TITLE
MCP Mods

### ISSUE THREAD

**Issue by ghost** [Hexeption/MCP-Reborn#264]
Hello

I would like that when you launch the minecraft client in 1.19.3, its launches in versions with forge to be able to put several mods
How is it possible to achieve this?

**Comment by PSJahn** [comment]
Use Forge MDK instead of MCP.

**Comment by ghost** [comment]
Yes but I have to have the source code file of Minecraft because I want to customize my Launcher, so remove the multiplayer, remove the singleplayer etc and just put a Play button and it connects to my server and it must it starts with forge because it's a Moddée server

**Comment by LunaticWasTaken** [comment]
Short Answer: not possible

Long answer: Implement the Forge Mod-Loader into your Client yourself.
I have no idea how it works, and i also have no idea if Forge's License allows it should it go public, but that's the only Idea i could think of myself.

**Comment by APie357** [comment]
@MrPiano5454 you use mixins to hook the main menu w/ a mixin and override the method that draws the buttons on the main menu to only display a single button.
